cookbooks/aws-parallelcluster-platform/templates/nvidia-imex/nvidia-imex-config.erb

```diff
@@ -0,0 +1,217 @@
+#                          the size specified in LOG_FILE_MAX_SIZE
+#       Non-zero Integer - Log is rotated upto the number of times specified in LOG_MAX_ROTATE_COUNT,
+#                          after the size of the log file reaches the size specified in LOG_FILE_MAX_SIZE.
+#                          Combined IMEX log size is LOG_FILE_MAX_SIZE multipled by LOG_MAX_ROTATE_COUNT+1
+#                          Once this threshold is reached, the oldest log file is purged and reused.
+LOG_MAX_ROTATE_COUNT=3
+
+#   Description: Redirect all the logs to syslog instead of logging to file
+#   Possible Values:
+#           0  - No
+#           1  - Yes
+#   Default Value: 0
+LOG_USE_SYSLOG=1
+
+#   Description: daemonize IMEX on start-up
+#   Possible Values:
+#       0  - No (Do not daemonize and run IMEX as a normal process)
+#       1  - Yes (Run IMEX process as Unix daemon
+#   Default Value: 1
+DAEMONIZE=1
+
+#   Description: Network interface to listen for IMEX peer communication.
+#                OPTIONAL - empty value will determine the bind IP from the node config file.
+#   Possible Values:
+#           A valid IPv4 address
+#           A valid IPv6 address
+#           No value - Determine bind IP from the node configuration file.
+#   Default Value:
+BIND_INTERFACE_IP=
+
+#   Description: Starting TCP port number for IMEX peer communication
+#   Possible Values:
+#           Any value between 0 and 65535
+#   Default Value: 50000
+SERVER_PORT=50000
+
+#   Description: Name of file containing IP addresses of nodes
+#   Possible Values:
+#       Full path/filename string (max length of 256).
+#       Default Value: /etc/nvidia-imex/nodes_config.cfg
+IMEX_NODE_CONFIG_FILE=<%= node['cluster']['nvidia']['imex']['shared_dir'] %>/nodes_config.cfg
+
+#  Description: Name of the network interface used for communication.
+#               OPTIONAL - If empty, network interface will be determined by matching bind IP to
+#                          node configuration file.  Only necessary to configure if the bind IP
+#                          is IPv6 link-local and on multiple network interfaces.
+#  Possible Values:
+#      Interface names like eth0, ens32 .. etc
+#  Default Value:
+NETWORK_INTERFACE=
+
+#  Description:  Controls whether IMEX should complete initialization without establishing quorum
+#  Possible values:
+#       NONE: Do not wait for any quorum with other nodes.
+#   RECOVERY: In case of unsafe IMEX termination, wait until all nodes that had previously imported
+#             have connected, allowing them time to safely clean up any potentially hanging references
+#  Default value: RECOVERY
+IMEX_WAIT_FOR_QUORUM=RECOVERY
+
+#  Description:  Enable authentication and encryption between nodes.
+#  Possible Values:
+#      0:  Disable encryption and authentication
+#      1:  Enable encryption and authentication
+#  Default value: 0
+IMEX_ENABLE_AUTH_ENCRYPTION=0
```

Comment:
why not enabling encryption?

Reply:
I kept the file as it is when it is generated by nvidia-imex installation, plus I would need to setup other values for enabling the auth either through SSL/TLS and other options like the SCORE,etc as per https://docs.nvidia.com/multi-node-nvlink-systems/imex-guide/config.html. i think this can be done in the next iteration and make the minimal chnages for now

Reply:
I'd recommend in the future using encryption, but don't see it as a blocker as initial release is a guide vs full-implementation. More importantly, I'd recommend we let the customer know about how the IMEX-channels work so they understand in a single channel mode there is no protection across users on the same node.

Reply:
Just to be sure I wasnt planning on making changes for IMex channel 's in the next iteration either as that would be more of a customization w.r.t DirectoryServices.